build.sc

```diff
@@ -126,6 +126,7 @@ object Deps {
   val millModuledefs = ivy"${millModuledefsString}"
   val millModuledefsPlugin =
     ivy"com.lihaoyi:::scalac-mill-moduledefs-plugin:${millModuledefsVersion}"
+  val millScip = ivy"io.chris-kipp::mill-scip::0.3.4"
```

Comment:
As we're going to use the dependency in a normal Scala module (which does not define a Mill platform), we need to specify the platform suffix manually.

```suggestion
  val millScip = ivy"io.chris-kipp::mill-scip_mill0.11:0.3.4"
```